packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -275,3 +264,107 @@ describe('listAccountHistory', () => {
     expect(history.find((h) => h.type === 'AccountToUtxos' && h.amounts[0] === '-3.97000000@DFI')).toBeTruthy()
   })
 })
+
+describe('listAccountHistory for poolpair', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+  const createToken = createTokenForContainer(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await container.waitForWalletBalanceGTE(200)
+    const address = await container.call('getnewaddress')
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await createToken(address, 'DDAI', 2000)
+
+    await client.poolpair.createPoolPair({
+      tokenA: 'DFI',
+      tokenB: 'DDAI',
+      commission: 0,
+      status: true,
+      ownerAddress: await container.call('getnewaddress')
+    })
+    await container.generate(1)
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should show AddPoolLiquidity', async () => {
+    await client.poolpair.addPoolLiquidity({
+      '*': ['10@DFI', '200@DDAI']
+    }, await container.call('getnewaddress'))
+    await container.generate(1)
+
+    const histories = await client.account.listAccountHistory()
+
+    const records = histories.filter((h) => h.type === 'AddPoolLiquidity')
+
+    expect(records.length).toStrictEqual(2)
+
+    expect(records).toEqual(expect.arrayContaining([
+      expect.objectContaining({
+        amounts: expect.arrayContaining([
+          expect.stringContaining('DFI-DDAI')
+        ])
+      }),
+      expect.objectContaining({
+        amounts: expect.arrayContaining([
+          '-10.00000000@DFI',
+          '-200.00000000@DDAI'
+        ])
+      })
+    ]))
```

Comment:
this structure is much easier for me to understand from an outsider looking at the code for the first time.